pulsar-common/src/main/java/com/yahoo/pulsar/common/naming/DestinationDomain.java

```diff
@@ -16,5 +16,29 @@
 package com.yahoo.pulsar.common.naming;
 
 public enum DestinationDomain {
-    persistent
+    persistent("persistent"), non_persistent("non-persistent");
```

Comment:
> We already have it at NonPersistentTopics and tested with testcase.

But int that case it's part of the path, right? Like `http://localhost:8080/admin/non-persistent/.....`

The topic itself would be `non-persistent://prop/cluster/namespace/topic`.

I don't have preference for any of the 2 names, though. Just curious whether that is a valid protocol scheme.